net-print/libinklevel/libinklevel-0.9.3.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools multilib
```

Comment:
This is one that lingers:
```suggestion
inherit autotools
```

... it's because earlier EAPIs didn't provide e.g. get_libdir.

Reply:
yeah I must have forgotten to remove it while editing

Reply:
multilib is one of those stinky ones where I'm always suspicious when I see it, but it's so easily done, absolutely